index.html

```diff
@@ -1 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <div id="main">
 
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p class="description">Thank you for taking the time to help us improve the platform</p>
+
+        <div id="form-div">
+            <form id="survey-form">
```

Comment:
:smiley: :+1: Good work on the form but I also recommend You apply the POST method on the form, thereby providing a recipient site for example, Formspree.